packages/renderer/src/lib/ui/NavPage.svelte

```diff
@@ -5,19 +5,19 @@ export let searchEnabled = true;
 </script>
 
 <div class="flex flex-col w-full h-full shadow-pageheader">
-  <div class="flex flex-col w-full h-full pt-4">
-    <div class="flex pb-2">
+  <div class="flex flex-col w-full h-full pt-4" role="region" aria-label="{title}">
+    <div class="flex pb-2" role="region" aria-label="header">
       <div class="px-5">
-        <h1 aria-label="{title}" class="text-xl first-letter:uppercase">{title}</h1>
+        <h1 class="text-xl first-letter:uppercase">{title}</h1>
```

Comment:
any reason why it's removed ?

Reply:
I think that having `h1` element and its text in a header locator is enough to identify the element safely using its role+text. But also does no harm keeping it there.